prow/cmd/deck/main.go

```diff
@@ -98,7 +98,7 @@ func gatherOptions() options {
 var (
 	// Matches letters, numbers, hyphens, and underscores.
 	objReg              = regexp.MustCompile(`^[\w-]+$`)
-	staticFilesLocation = "static"
+	staticFilesLocation = "./static"
```

Comment:
I rebased and refactored this path to be consistent, previously we only had `./static` for the generic static files handler, not for the favicon. I think this will actually fix things. Keeping the other change though

Reply:
Fingers crossed!